docs/users/exchanges.md

```diff
@@ -0,0 +1,361 @@
+
+### Flow 
+1. User requests to do a transfer to the exchange from his Hermez account using some front-end.
+2. Exchange creates an L2 (internal) account on behalf of the user (`L2ExUser-ETH`) and provides the address via front-end. This account is controlled by the exchange. 
+```js
+  // create new bjj private key to receive user transactions
+  const pvtBjjKey = Buffer.allocUnsafe(32).fill("1");
+
+  // create rollup internal account from bjj private key
+  const resExchangeWallet = await hermez.HermezWallet.createWalletFromBjjPvtKey(pvtBjjKey);
+  const hermezExchangeUserWallet = resExchangeWallet.hermezWallet;
+
+  // share public bjj key with the user
+  console.log(`Transfer funds to this hermez address:\n   ${hermezExchangeUserWallet.publicKeyBase64}\n\n`); 
+```
+3. User does a L2 transfer from his account to the destination account using the web wallet
+```js
+  const infoAccountUser = (await hermez.CoordinatorAPI.getAccounts(hermezUserWallet.hermezEthereumAddress, [tokenERC20.id]))
+    .accounts[0];
+
+  const state = await hermez.CoordinatorAPI.getState();
+  const usdTokenExchangeRate = tokenERC20.USD;
+  const fee = usdTokenExchangeRate ? state.recommendedFee.createAccountInternal / usdTokenExchangeRate : 0;
+
+  // user creates transaction to deposit 10 ether into exchange account
+  // deposit 10 ether
+  const userDepositToExchange = hermez.Utils.getTokenAmountBigInt("10.0", 18);
+  const compressedUserDepositToExchange = hermez.HermezCompressedAmount.compressAmount(userDepositToExchange);
+  // the following transaction would:
+  // - create an account for the exchange in hermez network
+  // - transfer to exchange account 0.1 eth
+  const transferToExchange = {
+    from: infoAccountUser.accountIndex,
+    to: hermezExchangeUserWallet.publicKeyBase64,
+    amount: compressedUserDepositToExchange,
+    fee : fee
+  };
+  console.log("transferToExchange: ", transferToExchange, fee);
+  // send tx to hermez network
+  await hermez.Tx.generateAndSendL2Tx(transferToExchange, hermezUserWallet, tokenERC20);
+```
+4. Exchange monitors balance of the account `L2ExUser-ETH`, and once transfer is complete, exchange performs transfer from `L2ExUser-ETH` to `L2Ex-ETH` for 10 ETH.
+```js
+  const pollingExchangeAddr = true;
+  while (pollingExchangeAddr){
+    const accountExchangeInfo = await hermez.CoordinatorAPI.getAccounts(hermezExchangeUserWallet.publicKeyBase64, [tokenERC20.id]);
+    if (accountExchangeInfo.accounts.length === 0){
+      console.log("Waiting for user deposit to be forged...");
+      await sleep(10000);
+    } else {
+      console.log("<=== Received deposit from user ===>");
+      console.log(`accountExchangeInfo:\n ${accountExchangeInfo.accounts[0]}`);
+      break;
+    }
+  }
+
+  const infoAccountExchangeUser = (await hermez.CoordinatorAPI.getAccounts(hermezExchangeUserWallet.publicKeyBase64, [tokenERC20.id]))
+   .accounts[0];
+
+
+  // Transfer funds to main exchange account
```

Comment:
The following code is rendered well in web.
Delete one line and it will be rendered properly